### PR TITLE
Updated Locksmith deployment to utilize Docker

### PR DESCRIPTION
**Pull request by akeem**
# Description

This is a bit messy, but should unblock us for the time being and will provide a way forward.

The following relevant things occur in this change:
* Adds Dockerfile for Locksmith only, for use with EB single container deployment
* Removes migration file from previous paradigm
* Provides a base script to create an application, environment, and deployment

Items to be addressed at a later:

* Update CNAME to point to new environment
* Ensuring environment is added to appropriate security group
* Adjust notification bot to reference new config after cut over.

# Issues

Fixes #
Refs #

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [x] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by akeem** [comment]
@julien51 when you have a minute or two would you mind giving this a look and providing some feedback?

File: locksmith/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+# Adding python which is needed by some dependencies
+RUN apk add --no-cache \
+    python \
+    python-dev \
+    py-pip \
+    build-base \
+  && pip install virtualenv
+
+# Update npm version to use cpm ci
```
Comment (cnasc) [inline locksmith/Dockerfile]:
```suggestion
# Update npm version to use npm ci
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -270,15 +270,20 @@ jobs:
       DOCKER_REPOSITORY: unlockprotocol
     steps:
       - *cached-checkout
-      - run:
-          name: Set GIT_COMMIT_DESC
-          command: echo 'export GIT_COMMIT_DESC=$(git log --format=%B -n 1 "$CIRCLE_SHA1")' >> $BASH_ENV
       - run:
           name: Set IS_FORKED_PR
-          command: ./scripts/circleci/set-is-forked-pull-request.sh >> $BASH_ENV
+          command: scripts/circleci/set-is-forked-pull-request.sh >> $BASH_ENV
```
Comment (julien51) [inline .circleci/config.yml]: any reason you changed this `./`? (I do not think this is  a problem  as long as `set-is-forked-pull-request.sh` is executable  but wanted to make sure youve seen this?)

File: .circleci/config.yml [hunk 3]
```diff
@@ -394,7 +399,7 @@ workflows:
       - deploy-locksmith-beanstalk:
           filters:
             branches:
-              only: master
+              only: master
```
Comment (julien51) [inline .circleci/config.yml]: Remove space here?

File: .circleci/config.yml [hunk 4]
```diff
@@ -270,15 +270,20 @@ jobs:
...
+          working_directory: /
+          command: |
+            sudo apt-get -y -qq update
+            sudo apt-get install python-pip python-dev build-essential
+            sudo pip install --upgrade setuptools
+            sudo pip install awsebcli --upgrade
       - run:
           name: Deploy to to Beanstalk
-          command: locksmith/scripts/deploy-elasticbeanstalk.sh "$CIRCLE_SHA1" "$CIRCLE_BRANCH" "$IS_FORKED_PR" "$GIT_COMMIT_DESC"
+          command: locksmith/scripts/deploy-eb-docker.sh "$APPLICATION" "$ENVIRONMENT" "$DB_USERNAME" "$DB_PASSWORD" "$DB_NAME" "$DB_HOSTNAME" "$NODE_ENV" "$IS_FORKED_PR"
```
Comment (julien51) [inline .circleci/config.yml]: Could you still use a name that mentions eb?

File: .circleci/config.yml (hunk 2 above)
Comment (akeem) [reply .circleci/config.yml]: good catch, this is a cut and paste error

**Pull request merged**